src/frontend/gatsby-config.js

```diff
@@ -1,4 +1,6 @@
 const path = require('path');
+const { sha } = require('git-repo-info')();
```

Comment:
If we can't use this, let's revert to what you did before (even though zeit/gatsby/webpack are wrong!)

Reply:
Zeit just hasn't liked this pr very much since the start, the original way nor the way you suggested. Everything that has worked locally fails to build on zeit. Do we just abandon this pr?